peninah98/my-brand
# updated files

### Updated files

### What does this PR do?

This PR implements a landing page

### Description of Task to be completed?


Add homepage
Update about page

### How should this be manually tested?


Clone [Repo](https://github.com/peninah98/my-brand)
Checkout branch feat-implement-landing-page-
Open the index.html in your browser to check the new homepage


What are the relevant pivotal tracker stories?
 [Trello](https://trello.com/c/5b5UFg61/15-login)

#[UgxAvO0y]



Screenshots
N/A
